js/ui/overview.js

```diff
@@ -229,20 +229,8 @@ const Overview = new Lang.Class({
                                         opacity: 0 });
         this._overview.add_actor(this._panelGhost);
 
-        this._searchEntry = new St.Entry({ style_class: 'search-entry',
```

Comment:
That would be really nice, specially the upstream part. I'll keep it in mind for the next rebase, which I hope will be easier and leave more time for upstreaming and doing things in a better way.